BruTile/BruTile.csproj

```diff
@@ -1,11 +1,30 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
 	<PropertyGroup>
-		<TargetFrameworks>netstandard2.0;net6.0</TargetFrameworks>
+		<TargetFrameworks>netstandard2.0;netstandard2.1;net6.0;net7.0;net8.0</TargetFrameworks>
 		<GeneratePackageOnBuild>true</GeneratePackageOnBuild>
 		<PackageOutputPath></PackageOutputPath>
 		<Description>BruTile is a C# library for GIS tile services such as those of Bing maps and OpenStreetMap</Description>
 		<PackageTags>tiling gis osm geo</PackageTags>
 	</PropertyGroup>
 
+	<ItemGroup Condition="$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'netstandard2.1'))">
+		<PackageReference Include="Microsoft.XmlSerializer.Generator" Version="8.0.0">

```

Comment:
It adds Microsoft.XmlSerializer.Generator greater equal than netstandard2.1 which means
netstandard2.1
Net6
Net7
Net8
I did this because netstandrad2.0 is not supported by Microsoft.XmlSerializer.Generator and with netstandard2.1 even Xamarin.Forms on Android and iOS is supported

Reply:
I see, I misunderstood IsTargetFrameworkCompatible. It is mentioned just twice in the entire Microsoft docs.
- https://learn.microsoft.com/en-us/dotnet/core/compatibility/sdk/8.0/trimming-unsupported-targetframework
- https://learn.microsoft.com/en-us/dotnet/core/tools/sdk-errors/

I considered dropping netstandard2.0 but that would mean dropping support for .NET framework which I don't want to do right now. If netstandard2.0 is dropped I think we also should drop netstandard2.1 and make .net6 the lowest supported version.